shared/util/interaction-manager.native.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import {InteractionManager} from 'react-native'
+
+// Arbitrary number
+InteractionManager.setDeadline(10)
```

Comment:
This is different. from the [docs](https://facebook.github.io/react-native/docs/interactionmanager.html):

> A positive number will use setTimeout to schedule any tasks after the eventLoopRunningTime hits the deadline value, otherwise all tasks will be executed in one setImmediate batch (default).

and 

> By default, queued tasks are executed together in a loop in one setImmediate batch. If setDeadline is called with a positive number, then tasks will only be executed until the deadline (in terms of js event loop run time) approaches, at which point execution will yield via setTimeout, allowing events such as touches to start interactions and block queued tasks from executing, making apps more responsive.